### ISSUE TITLE
Support file-association plist elements

### ISSUE THREAD

**Issue by kingjon3377** [perdian/macosappbundler-maven-plugin#38]
In porting an application from Ant to Maven (and to more recent versions of Java), one feature of [the JarBundler Ant task](https://github.com/UltraMixer/JarBundler) that this Maven plugin seems to lack any equivalent of is the ability to set up file associations.

In the Ant `build.xml`, as a child of the `<jarbundler>` element, I can write something like `<documenttype name="My Custom App files" extensions="aaa bbb" role="Editor" />`, and looking at the plist file in the generated application bundle, that turns into

```
<key>CFBundleDocumentTypes</key>
<array>
  <dict>
    <key>CFBundleTypeName</key>
    <string>My Custom App files</string>
    <key>CFBundleTypeRole</key>
    <string>Editor</string>
    <key>CFBundleTypeExtensions</key>
    <array>
      <string>aaa</string>
      <string>bbb</string>
    </array>
  </dict>
</array>
```

But as far as I can tell, there's no way to put anything like this under the `<plist>` element in `pom.xml`, which would be a very desirable feature for packaging apps that are intended to serve as viewers or editors for files.

**Comment by perdian** [comment]
Sounds like a useful extension, will take a closer look at it. 👍

**Comment by perdian** [comment]
Completed and released to Maven central as [`1.18.0`](https://github.com/perdian/macosappbundler-maven-plugin/releases/tag/v1.18.0).
